docs/dev/tutorials/custom-paymaster-tutorial.md

```diff
@@ -56,32 +56,38 @@ mkdir contracts deploy
 5. Create the file `hardhat.config.ts` and add the following:
 
 ```ts
+import { HardhatUserConfig } from "hardhat/config";
+
 import "@matterlabs/hardhat-zksync-deploy";
 import "@matterlabs/hardhat-zksync-solc";
 
-module.exports = {
+const config: HardhatUserConfig = {
   zksolc: {
-    version: "1.3.9",
+    version: "1.3.10", // Use latest available in https://github.com/matter-labs/zksolc-bin/
     compilerSource: "binary",
     settings: {},
   },
   defaultNetwork: "zkSyncTestnet",
-
   networks: {
+    hardhat: {
+      zksync: false,
+    },
     zkSyncTestnet: {
-      url: "https://testnet.era.zksync.dev",
-      ethNetwork: "goerli", // the RPC URL of the network (e.g. `https://goerli.infura.io/v3/<API_KEY>`)
+      url: "https://zksync2-testnet.zksync.dev",
```

Comment:
?

Reply:
fixed in https://github.com/matter-labs/zksync-web-era-docs/pull/465/commits/3ae8c68a33e059a1bf7fdf6f4a84ae2185722510